core-service/src/main/java/com/crmfoodestablishment/coreservice/dto/DishDto.java

```diff
@@ -1,16 +1,22 @@
 package com.crmfoodestablishment.coreservice.dto;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
 import java.math.BigDecimal;
 
 @Data
```

Comment:
do u really need Data here? maybe it's better to use only annotation u need separately

Reply:
Changed the annotation, only those that I need.